src/main/java/interfaces/IDatabase.java

```diff
@@ -1,7 +1,10 @@
 package interfaces;
 
-public interface IDatabase {
+import org.json.simple.JSONObject;
 
-    public String fetchRawData(String category);
+public interface IDatabase {
 
+    JSONObject database = new JSONObject();
+    default JSONObject getInfo(JSONObject database, String key) {return (JSONObject) database.get(key);}
+    default boolean keyExists(String key) {return database.containsKey(key);}
```

Comment:
These default methods seem to be redundant. There is no need to have default methods when you only have one Database class that is implementing this interface.